airbyte-config/persistence/src/main/resources/airbyte_configs_table.sql

```diff
@@ -0,0 +1,23 @@
+-- extensions
+ CREATE
+    EXTENSION IF NOT EXISTS "uuid-ossp";
+
+-- tables
+ CREATE
+    TABLE
+        IF NOT EXISTS AIRBYTE_CONFIGS(
+            id BIGINT GENERATED BY DEFAULT AS IDENTITY PRIMARY KEY,
+            config_id VARCHAR(36) NOT NULL DEFAULT uuid_generate_v4(),
+            config_type VARCHAR(60) NOT NULL,
+            config_blob JSONB NOT NULL,
+            created_at TIMESTAMPTZ NOT NULL DEFAULT CURRENT_TIMESTAMP,
+            updated_at TIMESTAMPTZ NOT NULL DEFAULT CURRENT_TIMESTAMP
+        );
+
+-- indices
+ CREATE
```

Comment:
So far every programmatic query includes the `config_type` column together with the `config_id` column. So the composite index (`config_type`, `config_id`) is sufficient. But it does not hurt to create a `config_id` only index, since that can accelerate manual queries. Will add it.